eox_nelp/edxapp_wrapper/backends/course_creators_l_v1.py

```diff
@@ -0,0 +1,23 @@
+"""Backend for course_creators module.
```

Comment:
the convention is `file_name_{first letter of the version}_v1.py ` so for this case this should be `course_creators_k_v1.py`

Reply:
Actually, I put lilac cause is like a reference. Most of the eox-core backends don't have the koa version, and instead, use the lilac. That is the reason I set Lilac.
![image](https://user-images.githubusercontent.com/51926076/185712745-f2fe4fa5-bb41-4890-9cde-a9820196cf67.png)

Reply:
They didn't test eox-core with koa so I think that is the reason. Anyway you can not ensure that this backend will work in lilac therefore you should set the name with the version that you use

Reply:
OK, you have a good point. Let me change it.

Reply:
Changed in  71e8c18e9cd5977ac38a022e1f2387d45ab1d9a1